modules/@apostrophecms/doc-type/index.js

```diff
@@ -832,7 +832,7 @@ module.exports = {
 
         project: {
           finalize() {
-            let projection = query.get('projection') || {};
+            let projection = query.get('project') || {};
```

Comment:
Why change `projection` to `project` while using `publicApiProjection`?

Reply:
MongoDB uses `project` for its own cursors when calling like this:

```
db.collection.find({}).project({ title: 1 }).sort({ createdAt: 1 }).toArray();
```

They do that because chainable methods read better as verbs.

A2 has a chainable `projection()` method for its query builders, which differs from MongoDB. So I changed it to match MongoDB in 3.x.

However `publicApiProjection` is not a verb, it is a noun ("this is the projection I would like used for public APIs"). I think it reads correctly as "Projection."

I would like to know what @abea thinks.

Reply:
(My original decision to go with `project` for the query builder method to match MongoDB was made a while ago, but in this PR I uncovered several bugs in that transition, which is how it came up for Anthony in his review.)

Reply:
What are other examples that would be used like this? If we're consistent using verb forms then it makes sense to go with MongoDB's pattern. The style and pattern would need to be documented ASAP to help keep us consistent.

Reply:
It would obviously need to be on the list for migration steps.